### ISSUE TITLE
frontend failed to build

### ISSUE THREAD

**Issue by nihalgurjar47** [Open-Source-Legal/OpenContracts#87]
When i am run to this command for production build **docker-compose -f local.yml --profile fullstack up** so i am getting error please check the below screenshot.
![Screenshot from 2023-03-02 14-13-53](https://user-images.githubusercontent.com/104414738/222377971-90974bda-2b82-4c50-80fa-0063c56424f7.png)

**Comment by JSv4** [comment]
Sorry, I'm so used to running this on my machine. I'll add a instruction to copy and configure the frontend.env file as .env from the [sample env directory](https://github.com/JSv4/OpenContracts/tree/main/docs/sample_env_files/frontend/local). Use the one called `django.auth.env` and just copy it to your frontend dir as .env to use the django auth. Use with.auth0.env if you want to configure auth0.

**Comment by JSv4** [comment]
I have to make one more change tonight. Will let you know when you can pull the latest version and run it with the django auth.

**Comment by JSv4** [comment]
@nihalgurjar47, updated the quickstart guide and also made some changes to the code that toggles between Django auth and auth0. I always use auth0 and some of the Django auth code had not been updated. The new quickstart *should* get you up and running. Sorry if I'm still overlooking any steps.

**Comment by nihalgurjar47** [comment]
Hi @JSv4 i am following your **quick-start** to start the project on local, but i am facing issue multiple point that I mention  below.
**### Step1-**
When i am start  project on local by using this command **docker-compose -f local.yml --profile fullstack up** i am getting this error.
![Screenshot from 2023-03-03 16-57-59](https://user-images.githubusercontent.com/104414738/223032173-43ebec6d-b4fb-46ce-857a-2c99910881ec.png)
To solve the above error I do the changes in directory name  From **.envs/.test/.django** To **.envs/.local/.django** then again start the project on local.
Note:- I checked your **local.yml** file, in this file you set env_file: for django **- ./.envs/.local/.django** and for postgres - **./.envs/.local/.postgres**

**### Step2 - Logging issue**
When I am trying to login through the UI, I passed  **username = admin** and **password=Openc0ntracts_def@ult** (mention in your .envs/.test/.djnago file ) I got this error.
![Screenshot from 2023-03-03 17-51-24](https://user-images.githubusercontent.com/104414738/223034755-2208853f-8fd4-4b6b-b49e-33d8c1c0bda2.png)
 I also created a new superuser through the command **docker-compose -f local.yml run django python manage.py createsuperuser** but I am still  facing  same issue.
![Screenshot from 2023-03-03 17-48-43](https://user-images.githubusercontent.com/104414738/223035728-15589ac6-94c8-4a70-bfbb-d567c3df049a.png)
![Screenshot from 2023-03-03 17-56-10](https://user-images.githubusercontent.com/104414738/223035849-cb7e5097-16af-47b5-ad60-a891668b4497.png)
**### Step3- database configuration DATABASE URL**
I checked database configuration but i got some configuration error like when django server up it's find the database configuration through the environment file, but some error occure like DATABASE URL is not found and I checked superuser entry is not insertrd in user table.
![Screenshot from 2023-03-06 12-35-55](https://user-images.githubusercontent.com/104414738/223041448-f01e1d11-afe1-4f2c-8576-ef3edfda5e15.png)

**Comment by nihalgurjar47** [comment]
Hi **@JSv4** thanks for the update it's **working** on my local..

**Comment by nihalgurjar47** [comment]
Hi @JSv4 
How to merged JSON generate of multiples label's?

Example: Document uploaded on opencontract server after that set a label text and added document in corpus (in case we set a multiple label for a single document) than we extracted the text from the document,I saw opencontract server generate particular label wise JSON. I simply wanted to know how to combine all JSON into a single document.
